docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+---
+---
+# Kubernetes Federated Services and Cross-Cluster Service Discovery
+
+This guide explains how to use Kubernetes Federated Services to deploy a common Service across multiple Kubernetes clusters to achieve cross-cluster service discovery and availibility zone fault tolerance for your Kubernetes applications.
```

Comment:
I think this sentence is too long to comprehend. I lost track at "common Service across multiple Kubernetes clusters". That might be also be a good checkpoint to break into a new sentence.